### ISSUE TITLE
White border in different dialogs

### ISSUE THREAD

**Issue by michaelhq** [obelisk79/OpenTheme#108]
Thank you for your great work!

Windows 11 with FreeCad 0.21.2

Perhaps it affects only FreeCAD on Windows: Using the dark theme, In some dialog boxes, a white border is shown on the bottom and right boarder. It stacks up when having nested dialogs.

![grafik](https://github.com/user-attachments/assets/59d1ea4a-a45c-4ca6-89c3-3c6c8c74e061)

![grafik](https://github.com/user-attachments/assets/952b66ba-0fd3-419f-bb8c-c878258a8b13)

**Comment by obelisk79** [comment]
That is certainly problematic. If you don't mind an experiment, can you test with a recent development build of FreeCAD and see if the problem is still there for you?

**Comment by michaelhq** [comment]
Installed the latest weekly build on a fresh windows 11.
With the latest weekly build, I have white boarder too. Not in the tree view, but in the properties dialog and on the customize dialog.

![grafik](https://github.com/user-attachments/assets/57501c87-db50-4811-a50c-6ae885708a9d)

![grafik](https://github.com/user-attachments/assets/eb635e31-1b6e-4907-a1ce-61bf49751027)

**Comment by obelisk79** [comment]
Yeah, I think these may be specific to windows, and specifically recent builds of FreeCAD on windows because I don't see the same strangeness with scroll bars or borders here.

![image](https://github.com/user-attachments/assets/30353681-9bf1-4df0-9c47-5514f47fbb15)


I'll see what I can do, but I don't have any computers with windows 11. Perhaps I can set up a VM for testing. Last time I tried though it was problematic.

**Comment by obelisk79** [comment]
I notice a few other sizing differences for you, do you have any kind of HiDPI or UI scaling enabled?
